Source/Blazorise.Bootstrap/wwwroot/blazorise.bootstrap.css

```diff
@@ -156,6 +156,11 @@
   max-height: var(--autocomplete-menu-max-height, 200px);
   overflow-y: scroll;
 }
+.b-is-autocomplete .dropdown-menu .dropdown-item.focus {
```

Comment:
This is currently part of BS only. What about other providers?

Reply:
Right, need to test and adapt other providers to do the same. Have yet to start that.